.github/workflows/tso-function-test.yaml

```diff
@@ -3,9 +3,11 @@ on:
   push:
     branches:
       - master
+      - release-5.*
```

Comment:
Should it be 5.2?

Reply:
It will only affect the current branch. So it's ok to use `5.*` here